cmd2/utils.py

```diff
@@ -1054,13 +1060,13 @@ def categorize(func: Union[Callable, Iterable[Callable]], category: str) -> None
         for item in func:
             setattr(item, constants.CMD_ATTR_HELP_CATEGORY, category)
     else:
-        if inspect.ismethod(func):
-            setattr(func.__func__, constants.CMD_ATTR_HELP_CATEGORY, category)
+        if inspect.ismethod(func) and hasattr(func, '__func__'):
```

Comment:
Why this extra check?

Reply:
I can take that out. I was trying to get around a mypy error and ended up just telling it to ignore it.